samcli/commands/build/command.py

```diff
@@ -136,6 +136,15 @@
     help="Enabled parallel builds. Use this flag to build your AWS SAM template's functions and layers in parallel. "
     "By default the functions and layers are built in sequence",
 )
+@click.option(
+    "--mount-with",
+    "-mw",
+    type=click.Choice(["READ", "WRITE"], case_sensitive=False),
```

Comment:
I would recommend writing an enum and comparing its value rather than keeping constant string (`"read", "write"`) in the code.